src/openrct2-ui/windows/Ride.cpp

```diff
@@ -1877,29 +1878,12 @@ namespace OpenRCT2::Ui::Windows
 
         static constexpr StringId GetRideTypeNameForDropdown(ride_type_t rideType)
         {
-            switch (rideType)
-            {
-                case RIDE_TYPE_1D:
-                    return STR_RIDE_NAME_1D;
-                case RIDE_TYPE_1F:
-                    return STR_RIDE_NAME_1F;
-                case RIDE_TYPE_22:
-                    return STR_RIDE_NAME_22;
-                case RIDE_TYPE_50:
-                    return STR_RIDE_NAME_50;
-                case RIDE_TYPE_52:
-                    return STR_RIDE_NAME_52;
-                case RIDE_TYPE_53:
-                    return STR_RIDE_NAME_53;
-                case RIDE_TYPE_54:
-                    return STR_RIDE_NAME_54;
-                case RIDE_TYPE_55:
-                    return STR_RIDE_NAME_55;
-                case RIDE_TYPE_59:
-                    return STR_RIDE_NAME_59;
-                default:
-                    return GetRideTypeDescriptor(rideType).Naming.Name;
+            auto stringId = GetRideTypeDescriptor(rideType).Naming.Name;
+            if (stringId == STR_UNKNOWN_RIDE)
+            {
+                stringId = STR_RIDE_NAME_UNKNOWN_INT32;
             }
```

Comment:
Not sure what the future plan for these 'unknown' ride types is, but would it be nicer to check if `kDummyRTD` is used here, or `.Category == RideCategory::none`?